plotting/quicklook_generator.py

```diff
@@ -405,12 +405,148 @@ def ultra_general_rates_2_plot(self) -> None:
+
+        fig, axes = plt.subplots(2, 1, figsize=(12, 8), sharex=True, sharey=True)
+
+        # Plot SW
+        pcm1 = axes[0].pcolormesh(
+            x_values_dt, energy_labels, sw_counts.T, shading="auto", cmap="viridis"
+        )
+        axes[0].set_title("SWP PCEM Count Rate")
+        axes[0].set_ylabel("Energy per charge (eV/q)")
+        fig.colorbar(pcm1, ax=axes[0], label="Counts")
+
+        # Plot PUI
+        pcm2 = axes[1].pcolormesh(
+            x_values_dt, energy_labels, pui_counts.T, shading="auto", cmap="viridis"
+        )
+        axes[1].set_title("SWP SCEM Count Rate")
+        axes[1].set_xlabel("Time (UTC)")
+        axes[1].set_ylabel("Energy per charge (eV/q)")
+        fig.colorbar(pcm2, ax=axes[1], label="Counts")
+
+        plt.tight_layout()
+        plt.show()
+
+    def swapi_absolute_detection_efficiency(self) -> None:
+        """Graph SWAPI absolute detection efficiency."""
+        # coincidence^2 / (primary*secondary)
+        if self.data_set is None:
+            raise ValueError("Must load in a dataset.")
+
+        # Time data
+        x_values = self.data_set["epoch"].values
+        x_values_dt = convert_j2000_to_utc(x_values)
+
+        # Get count rate data
+        sw_rate = self.data_set["swp_pcem_rate"]
+        sw_total_counts = sw_rate.sum(dim="energy")
+
+        pui_rate = self.data_set["swp_scem_rate"]
+        pui_total_counts = pui_rate.sum(dim="energy")
+
+        coin_rate = self.data_set["swp_coin_rate"]
+        coin_total_counts = coin_rate.sum(dim="energy")
+
+        denom = sw_total_counts * pui_total_counts
+        line = (coin_total_counts**2) / (denom)
+        line = line.where(denom != 0)
+
+        plt.plot(x_values_dt, line)
+        plt.title("SWAPI Absolute Detection Efficiency")
+        plt.xlabel("Time (UTC)")
+        plt.ylabel("Coincidence² / (Primary × Secondary)")
+        plt.grid(True)
+        plt.tight_layout()
+        plt.show()
+
+    def swapi_count_line(self) -> None:
+        """Generate SWAPI count rates LINE plot."""
+
+        if self.data_set is None:
+            raise ValueError("Must load in a dataset.")
+
+        # Time data
+        x_values = self.data_set["epoch"].values
+        x_values_dt = convert_j2000_to_utc(x_values)
+
+        # Get count rate data
+        # TODO: Ensure energy=0 is correct
+        sw_rate = self.data_set["swp_pcem_rate"]
+        sw_rate_single = sw_rate.isel(energy=0)
+        # sw_total_counts = sw_rate.sum(dim="energy")
+        pui_rate = self.data_set["swp_scem_rate"]
+        pui_rate_single = pui_rate.isel(energy=0)
+        # pui_total_counts = pui_rate.sum(dim="energy")
+        coin_rate = self.data_set["swp_coin_rate"]
+        coin_rate_single = coin_rate.isel(energy=0)
+
+        fig, axes = plt.subplots(3, 1, sharex=True, sharey=True)
+
+        # Plot SW
+        axes[0].plot(x_values_dt, sw_rate_single, color="blue")
+        axes[0].set_title("SW PCEM Total Count Rate")
```

Comment:
Can you spell out all places where there are `SW` or `SWP` to `SWAPI` because we don't want to confuse with past mission data plots or solar wind abbrevation.
```suggestion
        axes[0].set_title("SWAPI PCEM Total Counts")
```